classes/cf-input.class.php

```diff
@@ -271,6 +271,17 @@ function __construct( $conf ) {
 			$this->value = $this->get_value();
 			return true;
 		}
+
+		/**
+		 * Added support for php 5.3-
+		 *
+		 * @param $conf
+		 *
+		 * @return mixed
+		 */
+		function cf_input( $conf ) {
```

Comment:
Missing function visibility. I know it's `public` by default, we should make that explicit.